### PR TITLE
#36 set up angular and add first component with data-service, add rou…

### PR DESCRIPTION
**Pull request by ElenaSchnedlitz**
SetUp Angular Project

# 💻 What
 setup Angular-Project with first component and basic data-service for further use

# ❓ Why
to start web implementation

# 📘 How
vhange angular version -> npm install, generate component (ng g c "name") same with service (ng g service "name"), add formGroup and FormControls to component 

# 👀 See

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -2,13 +2,27 @@ import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
 
 import { AppComponent } from './app.component';
+import { LoginComponent } from './login/login.component';
+import {ReactiveFormsModule} from "@angular/forms";
```
Comment (Ihret) [inline src/app/app.module.ts:6]: test

**Review by Ihret — APPROVED** [review]
Rakete

**Review by kaulex99 — APPROVED** [review]
LGTM 🚀

**Review by DanielZellner — APPROVED** [review]
Looks good to me :)

File: src/app/app.module.ts [hunk 2]
```diff
@@ -2,13 +2,27 @@ import { NgModule } from '@angular/core';
...
+import {HttpClientModule} from "@angular/common/http";
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    LoginComponent
   ],
   imports: [
-    BrowserModule
```
Comment (Ihret) [inline src/app/app.module.ts:11]: bitte add wieder Browsermodule oder delete den import für das Browsermodule

**Review by Ihret — APPROVED** [review]

**Pull request merged**